tests/functional/backend/common.py

```diff
@@ -31,7 +31,12 @@ def setUpClass(cls):
         cls.deployment_stage = os.environ["DEPLOYMENT_STAGE"]
         cls.config = CorporaAuthConfig()
         cls.session = requests.Session()
-        retry_config = Retry(total=7, backoff_factor=2, status_forcelist=[500, 502, 503, 504])
+        retry_config = Retry(
+            total=7,
+            backoff_factor=2,
+            status_forcelist=[500, 502, 503, 504],
+            allowed_methods=frozenset({"DELETE", "GET", "HEAD", "OPTIONS", "PUT", "TRACE", "POST"}),
```

Comment:
Would add a comment, similar to this PR's description, justifying why its okay to include the non-idempotent POST here.  Also worth noting that we're just adding POST to Retry's DEFAULT_ALLOWED_METHODS.  Could write that as `DEFAULT_ALLOWED_METHODS.add("POST")`, but not sure that's better.

Reply:
You don't need frozenset. Let's only keep the methods we actually use to avoid creating confusion. Remove `TRACE` or `OPTIONS`.
```suggestion
            allowed_methods={"DELETE", "GET", "HEAD", "PUT" "POST"},
```